apps/site/src/app/(main)/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -21,33 +18,35 @@ export default async function Sponsors() {
 				Sponsors
 			</h2>
 			<p className="max-w-md mb-12">
-				Interested in sponsoring IrvineHacks 2024? Check out our
-				information above to learn more about our event! For more
-				information, please email us at{" "}
-				<a
-					href="mailto:hack@uci.edu"
-					className="hover:underline font-bold"
-				>
+				Interested in sponsoring IrvineHacks 2024? Check out our information
+				above to learn more about our event! For more information, please email
+				us at{" "}
+				<a href="mailto:hack@uci.edu" className="hover:underline font-bold">
 					hack@uci.edu
 				</a>
 				.
 			</p>
-			<div className="grid grid-cols-1 md:grid-cols-2 gap-14 items-center mb-12">
-				{sponsors.sponsors.map(({ _key, name, url, logo }) => (
-					<a
-						key={_key}
-						href={url}
-						target="_blank"
-						rel="noopener noreferrer"
-					>
-						<img
-							className="w-full"
-							src={builder.image(logo).format("webp").url()}
-							alt={name + " logo"}
-						/>
-					</a>
-				))}
-			</div>
+			<SponsorTier
+				sponsors={sponsors.get("platinum")}
+				className={styles.platinum}
+			/>
```

Comment:
Suggestion: seems like we're repeating instances of this component a lot. Could we use an array of tiers and map them to each instance to reduce duplication?